app/src/templates/quiz.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base.html" %} {% block title %}Quiz{% endblock %} {% block styling %}
+<!-- Local stylesheets -->
+<link rel="stylesheet" href="{{ url_for('static', filename='css/quiz.css') }}" />
+{% endblock %} {% block content %}
+<main>
+    <div id="heading">
+        <h1>Name: {{ quiz.title }}</h1>
+    </div>
+
+    <div id="quiz-container">
+        <div id="quiz-heading">
+            <div id="progress-container">
+                <p class="count-box" id="selectedCount">Question: 0</p>
```

Comment:
Small suggestion, I think it'd be more useful for the user to see the number of questions they have left to answer rather than the number of questions they've answered so far, but it's up to you